apiary.apib

```diff
@@ -3250,6 +3250,357 @@ Update the current timer. Only possible if there is a timer running.
 
 + Response 204
 
+# Group Work Orders
+
+## Work Orders [/workOrders]
+
+### workOrders.draft [POST /workOrders.draft]
+
+Add a draft work order.
+
++ Request (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + customer (object, required)
+            + type (enum[string], required)
+                + Members
+                    + contact
+                    + company
+            + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + performed_at: `2017-05-26T10:01:49+00:00` (string, required)
+        + description: `Pipework intervention` (string, optional)
+        + department_id: `cef01135-7e51-4f6f-a6eb-6e5e5a885ac7` (string, optional)
+        + layout_id: `a05320f6-b3ef-4ff9-a287-2074a12fb3e9` (string, optional)
+        + milestone_id: `e94c1363-3426-46a4-921d-fce03f39bbd5` (string, optional)
+        + event_id: `9fc14045-5c6c-4ba8-8672-42ea3f26aa63` (string, optional)
+        + intervention_address (object, optional)
+            + Include Address
+            + customer (object, optional) - Customer this address refers to.
+                + type (enum[string], required)
+                    + Members
+                        + contact
+                        + company
+                + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
+        + distance_travelled (object, optional)
+            + type: `km` (enum[string], required)
+                + Members
+                    + km
+            + value: `25.2` (number, required)
+        + assignees (array, required)
+            + (object)
+                + user_id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string, required)
+                + hourly_rate (Money, optional)
+                + work_hours (array, optional)
+                    + (object)
+                        + started_at: `2017-04-26T10:01:49+00:00` (string, required)
+                        + duration: 3600 (number, required) - In seconds
+                        + description (string, optional)
+        + products (array, optional)
```

Comment:
I guess some customers also have services as "products". But not sure if there's a good name that combines something like material/services.